modules/administration-guide/pages/limiting-the-number-of-workspaces-that-a-user-can-run-simultaneously.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: PROCEDURE
+:navtitle: Limiting the number of workspaces that a user can run simultaneously
+:description: Limiting the number of workspaces that a user can run simultaneously
+:keywords: administration guide, number, workspaces
+
+[id="limiting-the-number-of-workspaces-that-a-user-can-run-simultaneously_{context}"]
+= Limiting the number of workspaces that a user can run simultaneously
+
+Users can run multiple workspaces with different configurations simultaneously. To reduce demand on the cluster, you can limit the number of workspaces that each user can run simultaneously.
```

Comment:
By default, this number is 1. So the point of this article is to explain how to INCREASE capactity per user.

Reply:
Fixed. Thank you for clarifying this, Nick.